.github/workflows/test_precommit.yml

```diff
@@ -1,6 +1,10 @@
 name: test_precommit
 permissions: read-all
-on: pull_request
+on:
+  pull_request:
+  merge_group:
+    branches:
+      - develop
```

Comment:
Are we going to rename master to develop?

Reply:
Nope, that's a typo